packages/php-wasm/compile/php/Dockerfile

```diff
@@ -723,6 +723,10 @@ RUN set -euxo pipefail; \
 	source /root/emsdk/emsdk_env.sh; \
 	export EXPORTED_FUNCTIONS=$'["_exit", \n\
 "_php_wasm_init", \n\
+"_malloc", \n\
+"stringToUTF8", \n\
+"UTF8ToString", \n\
+"lengthBytesUTF8", \n\
```

Comment:
Is this export required? These are listed in `EXTRA_EXPORTED_RUNTIME_METHODS` below.